graylog2-web-interface/src/components/grok-patterns/GrokPatterns.jsx

```diff
@@ -97,6 +97,7 @@ const GrokPatterns = createReactClass({
                               name={pattern.name}
                               pattern={pattern.pattern}
                               testPattern={this.testPattern}
+                              patterns={this.state.patterns}
```

Comment:
We should filter the current pattern being edited by the user, as that should not be added to the Grok pattern.